NetKet/Graph/custom_graph.hpp

```diff
@@ -56,6 +58,16 @@ class CustomGraph : public AbstractGraph {
         std::vector<std::vector<int>> edges =
             pars["Graph"]["Edges"].get<std::vector<std::vector<int>>>();
         AdjacencyListFromEdges(edges);
+
+        // Specify edge colors if there is more than one
+        if (FieldExists(pars["Graph"], "EdgeColors")) {
```

Comment:
We enter here only if "Edges" are defined in the input. What about if AdjacencyList is defined instead? I guess we should call `EdgeColorsFromList` in that case too